docs/guide.md

```diff
@@ -56,7 +56,11 @@ webdriverio
     .end();
 ```
 
-** 6. Run your test file**
+** 6. Install browser and appropriate driver**
+
+In the example above FireFox is specified as the desired browser, but may not be currently installed on the testing system. The choice is to either install FireFox or download [Third Party Drivers](http://docs.seleniumhq.org/download/#thirdPartyDrivers) and place them in the same directory as the selenium standalone server.
```

Comment:
I suppose there are two ways of looking at it either selenium standalone server can load the web driver or the driver can be run standalone.  If the goal of the guide is to have users run with FireFox then they are required to have it installed.  In the other case, assuming they want to utilize WebdriverIO, they should download a web driver and have selenium standalone server take care of running it.